OWNERS.md

```diff
@@ -0,0 +1,25 @@
+# AIP Maintainers
+
+This page lists all active maintainers of the aip.dev and standard. Maintainers have write and approval privileges to the aip.dev repository.
+
+## Maintainers
+
+| Name                                                | GitHub                                           | Organization (if applicable) |
+| --------------------------------------------------- | ------------------------------------------------ | ---------------------------- |
+| [Yusuke Tsutsumi](mailto:yusuketsutsumi@google.com) | [@toumorokoshi](https://github.com/toumorokoshi) | Google                       |
+| [Mak Ahmad](mailto:mak1@fb.com)                     | [@makahmad](https://github.com/makahmad)         | Meta                         |
+| Richard Frankle                                     | [@rofrankel](https://github.com/rofrankel)       | Roblox                       |
```

Comment:
```suggestion
| Richard Frankel                                     | [@rofrankel](https://github.com/rofrankel)       | Roblox                       |
```

Reminder that committing this via GH will create CLA issues; you might want to replicate this fix yourself.